src/Azs.Subscriptions.Admin/custom/New-AzsOffer.ps1

```diff
@@ -179,7 +179,18 @@ param(
         if ( $PSCmdlet.ParameterSetName -eq 'CreateExpanded' )
         {
             # Autorest generated code doesn't throw error in case resource already exists
-            $resource = Get-AzsAdminManagedOffer -Name $Name -ResourceGroupName $ResourceGroupName -ErrorAction SilentlyContinue
+            $params = $PSBoundParameters + @{}
+            $params.Remove('AddonPlanDefinition')
```

Comment:
why do we need to remove the params here @viananth ?

Reply:
@bganapa I am trying to pass all the relevant parameters to "Get-AzsAdminManagedOffer" which were passed to "New-AzsOffer" here.
since 'AddonPlanDefinition' is not a parameter for "Get-AzsAdminManagedOffer" cmdlet, I am removing it. Similarly for others.

Reply:
@dolauli Here we have an issue with the default value of (Get-AzContext).SubscriptionId throwing an error while playing back the recorded files and we are in need of passing in all the params again. Is there a better way to handle this scenario? appreciate your inputs

Reply:
@bganapa updated the PR as per the suggestion.

Reply:
@bganapa please approve this